.circleci/config.yml

```diff
@@ -31,8 +31,8 @@ jobs:
                 -e NRN_RELEASE_UPLOAD \
                 -e SETUPTOOLS_SCM_PRETEND_VERSION \
                 -e NRN_BUILD_FOR_UPLOAD=1 \
-                'neuronsimulator/neuron_wheel:latest-gcc9-aarch64' \
-                packaging/python/build_wheels.bash linux << parameters.NRN_PYTHON_VERSION >> coreneuron
+                'neuronsimulator/neuron_wheel:latest-aarch64' \
```

Comment:
New Docker images have the tag `latest-[ARCH]` for consistency. Also, `manylinux2014` comes with GCC 10, not 9, so it was a misleading name anyway.